### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by PrantoshB**
## Hello There 👋
hello

In this branch, I have:
- Created Views as per the wireframes
- Styled the page using pure CSS
- Fixed Linters Errors
- Followed Git and GitHub Flows

### Screenshots:

- #### User Page
Screenshot from 2023-04-06 19-09-37
- #### User Posts
Screenshot from 2023-04-06 19-09-15
- #### Single Post
Screenshot from 2023-04-06 19-09-23

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,44 @@
...
+            <hr id="half">
+            <% end %>
+            <div class="comments-container">
+                <% @comments=Post.find(post.id).comments %>
+                    <% @comments.each do |comment| %>
+                        <div  id="single-comment"> 
+                            <span> <%= comment.user.name %> : <%= comment.text %></span>
+                        </div>
+                    <% end %>
+            </div>
```
Comment (gemmen29) [inline app/views/posts/index.html.erb:37]: - [ ] Kindly use the ` most_recent_comments` function here as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.). To match the wireframe.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @PrantoshB, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (PrantoshB) [reply app/views/posts/index.html.erb]: Thanks @gemmen29, for the detailed review. 🤝

I have refactored the code and submitting it for the next review. 💪

Cheers 🍻

**Review by DeliceLydia — APPROVED** [review]
## Hi @PrantoshB, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed all changes requested


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**